src/features/MentorPage/MentorCard/MentorCardHeaderBasicInfo.tsx

```diff
@@ -0,0 +1,84 @@
+import styled from 'styled-components';
+import { ListCardProps } from '../../../../../mentor_card/src/features/MentorPage/ListCard/types';
+import { Text } from '../../../components/CommonTextStyles/Text';
+import CSS from 'csstype';
+import MentorCardLanguages from './MentorCardLanguages';
+
+type Props = {
+  mentorCardData: ListCardProps;
+};
+
+const customStylus: CSS.Properties = {
+  margin: '0px',
+  display: 'flex',
+  flexWrap: 'wrap',
+};
+
+const truncateText: CSS.Properties = {
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  margin: '0px',
+  width: '100%',
+  fontFamily: '"Source Sans Pro"',
+  fontStyle: 'normal',
+  fontWeight: '400',
+  fontSize: '1rem',
+  lineHeight: '1.5rem',
+  color: '#ffffff',
+  textAlign: 'center',
+};
+
+const MentorCardHeaderBasicInfo = ({ mentorCardData }: Props) => {
+  if (mentorCardData !== undefined) {
+    const age = new Date().getFullYear() - mentorCardData.mentor.birthYear;
+    return (
+      <BasicInfoContainer>
+        <Text color="white" variant="h2" style={{ margin: 0 }}>
+          {mentorCardData.mentor.displayName}
+        </Text>
+        <NameDivider />
+        <Text color="white" variant="p" style={customStylus}>
+          {age} v. <StyledDivider>|</StyledDivider>{' '}
+          {mentorCardData.mentor.region}
+        </Text>
+        <Text color="white" variant="p" style={{ margin: 0 }}>
+          {mentorCardData.isLoggedIn}
+        </Text>
+        <p title={mentorCardData.contactMessage} style={truncateText}>
+          {mentorCardData.contactMessage}
+        </p>
+        <MentorCardLanguages languages={mentorCardData.mentor.languages} />
+      </BasicInfoContainer>
+    );
+  }
+  return <></>;
```

Comment:
Revert the condition and return null.

But this makes me think when we have so much of the same problem.. 
Maybe we can sharpen the Props a bit and then the mentorCardData is required value. Because it makes no sense to even show the card if there is no data?
You can check on the parent-level that the mentorCardData exists and then you dont have to check in every child that the data exists.